jscomp/frontend/ast_attributes.ml

```diff
@@ -373,3 +373,10 @@ let bs_return_undefined : attr =
           pstr_loc = locg;
         };
       ] )
+
+let is_gentype (attr : attr) =
+  match attr with
+  | {Location.txt = "genType" | "gentype"; _}, _ -> true
+  | _ -> false
+
+let gentype : attr = ({txt = "genType"; loc = locg}, Ast_payload.empty)
```

Comment:
looks like this is not used?

Reply:
ahhh right. I think it's better to use it instead of copying the original attrs

Reply:
done. can you push the changelog commit again? I forgot pull.